### ISSUE TITLE
Detect unreachable code in conditional expressions

### ISSUE THREAD

**Issue by RJ722** [jendrikseipp/vulture#178]
Extending the  discussion from https://github.com/jendrikseipp/vulture/issues/177#issuecomment-546272060, we would need a method to visit the `IfExp` node.

IMHO, we can define it the same way we define `visit_If`:

```python
def visit_IfExp(self, node):
    self._handle_conditional_node(node, 'ternary')
```

Then, we would also need to change `_handle_conditional_node` method:

```python
if utils.condition_is_always_false(node.test):
    # node.body in the case of ast.IfExp points out to the next instruction
    # In other cases, it's generally a list of instructions.
    last_node = node.body if isinstance(node, ast.IfExp) else node.body[-1]
    self._define(self.unreachable_code, name, node, last_node, # ...)
else:
    else_body = [node.orelse] if isinstance(node, ast.IfExp) else node.orelse
    # ...
```

**Comment by RJ722** [comment]
Hey, @jendrikseipp can you label this code as "Good First Issue", please? This way, GitHub will show it whenever someone opens https://github.com/jendrikseipp/vulture/contribute (which currently shows not so good first issues 😅).

**Comment by jendrikseipp** [comment]
Good idea!

**Comment by jendrikseipp** [comment]
Fixed in #207.
